Configuration/Settings.yaml

```diff
@@ -4,6 +4,6 @@ Neos:
       Array: 'TYPO3\Eel\Helper\ArrayHelper'
 
     metaDataMapping:
-      title: '${asset.Title || iptc.Title || exif.imageDescription}'
+      title: '${String.crop(asset.Title || iptc.Title || exif.imageDescription, 255)}'
       caption: '${asset.Caption || iptc.Description}'
-      tags: '${Array.concat(asset.tags, iptc.keywords)}'
\ No newline at end of file
+      tags: '${Array.concat(asset.tags || [], iptc.keywords || [])}'
```

Comment:
I think this wouldn't fix the problem as the problem occurs when a tag with an empty name is created.